app/javascript/hooks/request_query.js

```diff
@@ -0,0 +1,24 @@
+import { usePaginatedQuery, useQuery } from 'react-query'
+import { UrlParams } from '../utils/url_params'
+
+async function handleFetch(key, url, query) {
+  const resp = await fetch(`${url}?${new UrlParams(query).toString()}`)
+
+  return resp.json()
+}
+
+export function usePaginatedRequestQuery(key, request, fetch) {
```

Comment:
The `fetch` parameter is unused. Did you plan on using it somewhere? If not:

```suggestion
export function usePaginatedRequestQuery(key, request) {
```

Reply:
Thanks!